src/reset_atoms_image.cpp

```diff
@@ -93,24 +93,23 @@ void ResetAtomsImage::command(int narg, char **arg)
                                    "c_ifmax_r_i_f[*] c_ifmin_r_i_f[*]");
 
   // trigger computes
-  // need to ensure update->first_update = 1
-  //   to allow this input script command prior to first run/minimize
-  //   this is b/c internal variables are evaulated which invoke computes
-  //   that will trigger an error unless first_update = 1
-  // reset update->first_update when done
-
-  int first_update_saved = update->first_update;
-  update->first_update = 1;
-
+  // need to first initialize compute flags to allow this input script command prior
```

Comment:
I think here what is needed is to insure the created computes actually invoke their init() method
before their compute_peratom() or array() method.  The simplest way would be to create them
before the lmp->init() call above.  Otherwise, use frags->init() rather than frags->init_flags(),
since the latter doesn't matter and the computes will be deleted below.

Reply:
That does the trick. Done.